src/assets/i18n/en.json

```diff
@@ -498,6 +498,22 @@
     "eventClassifierCode": {
       "label": "Event Classifier Code",
       "select": "select event classifier code"
+    },
+    "portCall": {
+      "add": "Add a new Port Call",
+      "create": "Create Port Call",
+      "createWithTimestamp": "Create Port Call + Timestamp",
+      "label": "Port Calls",
```

Comment:
Maybe we should use "Port Visit" to avoid clashing with existing terms (some people also use port call for something which is sometimes a terminal call and sometimes not)

Reply:
(As a general "Port Call" -> "Port Visit" remark)